exercises/grep/description.md

```diff
@@ -1,77 +1,25 @@
-# Description
+# Instructions
 
-Search a file for lines matching a regular expression pattern. Return the line
-number and contents of each matching line.
+Search files for lines matching a search string and return all matching lines.
 
-The Unix [`grep`](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/grep.html) command can be used to search for lines in one or more files
-that match a user-provided search query (known as the *pattern*).
+The Unix [`grep`](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/grep.html) command searches for lines in files that match a regular expression. Your task is to implement a simplified `grep` function, which supports searching for literal strings.
 
-The `grep` command takes three arguments:
+The `grep` function takes three arguments:
 
-1. The pattern used to match lines in a file.
-2. Zero or more flags to customize the matching behavior.
-3. One or more files in which to search for matching lines.
+1. The string to search for.
+2. Zero or more flags to customize its behavior.
+3. One or more files to search in.
 
-Your task is to implement the `grep` function: given a list of files, find all
-lines that match the specified pattern.
-Return the lines in the order they appear in the files.
-You'll also have to handle options (given as flags), which control how matching
-is done and how the results are to be reported.
-
-As an example, suppose there is a file named "input.txt" with the following contents:
-
-```text
-hello
-world
-hello again
-```
-
-If we were to call `grep "hello" input.txt`, the result should be:
-
-```text
-hello
-hello again
-```
-
-If given multiple files, `grep` should prefix each found line with the file it was found in.
-As an example:
-
-```text
-input.txt:hello
-input.txt:hello again
-greeting.txt:hello world
-```
-
-If given just one file, this prefix is not present.
+It then reads the contents of the specified files (in the order specified),
+finds the lines that contain the search string,
+and finally returns those lines (in the order in which they were found).
 
 ## Flags
 
-As said earlier, the `grep` command should also support the following flags:
+The `grep` function should support the following flags:
 
-- `-n` Prefix each matching line with its line number within its file.
-  When multiple files are present, this prefix goes *after* the filename prefix.
-- `-l` Print only the names of files that contain at least one matching line.
-- `-i` Match line using a case-insensitive comparison.
+- `-n` Prepend the line number to each line in the output.
```

Comment:
What happened to the "multiple files" note?

Reply:
I've now included this information in line 14 and this flag's description.